### DISCUSSION TITLE
No response from support.

### DISCUSSION THREAD

**Discussion by teletvbisgotbanned** [community/community#121865] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have a ticket which is already open for a MONTH.
I got ZERO response from support which is crazy.

**Comment by ebndev** [comment]
Hi @teletvbisgotbanned,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion and any duplicate discussions.

We appreciate your understanding!
